src/website/views/custom-elements/expansion-panel.pug

```diff
@@ -0,0 +1,297 @@
+---
+title: Expansion panel
+scripts:
+  - js/ce/ux-chi-ce
+styles:
+  - assets/styles/expansion-panel-ce
+---
+
+p.-text
+  | Expansion panels are components designed to be folded and stacked in sequence. They have four states: active,
+  | pending, done and disabled. When pending, which is the default state, only the title is shown. When active, panels expand to show
+  | content or forms. When done, content or form data is presented in a read only state. When disabled, a muted title
+  | is shown in a non-clickable state.
+
+  .example.-mb3.-mt3
+    .-p3
+      chi-expansion-panel(step='1', heading='Done panel', state='done')
+        span(slot='done' class='-text') 
```

Comment:
@SergioQCL this span is causing a wrong height of 4px. Please change it to a div